rest/src/routes/nodeRoutes.js

```diff
@@ -127,11 +159,36 @@ module.exports = {
 			}));
 
 		server.get('/node/time', (req, res, next) => {
-			const packetBuffer = packetHeader.createBuffer(PacketType.timeSyncNodeTime, packetHeader.size);
-			return connections.singleUse()
+			const packetBuffer = packetHeader.createBuffer(
+				PacketType.timeSyncNodeTime,
+				packetHeader.size
+			);
+			return connections
+				.singleUse()
+				.then(connection => connection.pushPull(packetBuffer, timeout))
+				.then(packet => {
+					res.send(
+						buildResponse(packet, nodeTimeCodec, routeResultTypes.nodeTime)
+					);
+					next();
+				});
+		});
+
+		server.get('/node/unlockedaccount', (req, res, next) => {
```

Comment:
Do you need the node name? The current implementation will only query the API node.